src/web_app_skeleton/script/check_system

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+BOLD_RED_COLOR="\e[1m\e[31m"
+
+# Prints error and exit.
+# example:
+#   print_error "Redis is not running. Run it with some_command"
+print_error() {
+    printf "${BOLD_RED_COLOR}There is a problem with your system setup:\n\n   ${BOLD_RED_COLOR}$1 \n\n"
```

Comment:
I think passing in the text without this prefix is probably enough. Like "Redis isn't running" or "postgres isn't running. Please start it." is sufficient. Thoughts?

Reply:
You mean get rid of this top line "There is a problem...", and just leave the bottom part?

<img width="691" alt="Screen Shot 2020-03-27 at 1 14 32 PM" src="https://user-images.githubusercontent.com/2391/77796642-ecaf3a00-702c-11ea-8b83-3dbd8e5554a4.png">

Reply:
Never mind, I like that a lot :) I didn't realize it had some newlines! That looks super nice 👍

Reply:
I'm bad at reading shell scripts :)